src/FFmpegAudioDecoder.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// Audio resampling: https://ffmpeg.org/doxygen/2.5/resampling_audio_8c-example.html#_a21
 
-using FFmpeg.AutoGen;
+using FFmpeg.AutoGen.Abstractions;
```

Comment:
What's the difference between FFmpeg.AutoGen and FFmpeg.AutoGen.Abstractions? Is there an advantage here?

Reply:
The FFmpeg.AutoGen.Bindings.DynamicallyLoaded depends on FFmpeg.AutoGen.Abstractions so we have to use it instead of FFmpeg.AutoGen